### ISSUE TITLE
F821 for PEP695 ParamSpec

### ISSUE THREAD

**Issue by AlexanderPodorov** [PyCQA/pyflakes#797]
The following example from official python docs shows warnings on `flake8==6.1.0` (I believe `vscode-flake8` extension ships with this version):
See https://docs.python.org/3/library/typing.html#typing.ParamSpec
![image](https://github.com/PyCQA/pyflakes/assets/54511144/6f3f5e78-4ba3-4662-920c-ef5147afa84a)
See the original issue https://github.com/microsoft/vscode-flake8/issues/271 for reference.

**Comment by sigmavirus24** [comment]
Please do not post screenshots. They're not searchable or accessible. Can you confirm the version of pyflakes you're using with the vscode flake8 extension and which version of Python flake8/pyflakes is running in?

**Comment by AlexanderPodorov** [comment]
Hi,
I'm sorry about posting a screenshot, I did not realize that.
Microsoft's vscode `flake8` extension version is `v2023.11.13551006`.
It says it ships with `flake8==6.1.0`.
I can't check `pyflakes` version unfortunately, since the extension does not provide any info about it.
I filed this issue for `pyflakes` since `flake8` docs says the following:
> Error codes starting with F should be reported to [pyflakes](https://github.com/pyflakes/pyflakes)

Thanks!
